web/modules/custom/dpl_dashboard/src/Plugin/Block/DashboardBlock.php

```diff
@@ -66,78 +64,80 @@ public static function create(ContainerInterface $container, array $configuratio
+      'loans-overdue-text' => $this->t('Returned too late', [], ['context' => 'Dashboard']),
+      'loans-soon-overdue-text' => $this->t('To be returned soon', [], ['context' => 'Dashboard']),
+      'material-and-author-text' => $this->t('and', [], ['context' => 'Dashboard']),
+      'material-by-author-text' => $this->t('By', [], ['context' => 'Dashboard']),
+      'no-physical-loans-text' => $this->t('At the moment, you have 0 physical loans', [], ['context' => 'Dashboard']),
+      'no-reservations-text' => $this->t('At the moment, you have 0 reservations', [], ['context' => 'Dashboard']),
+      'pay-owed-text' => $this->t('Read more', [], ['context' => 'Dashboard']),
+      'physical-loans-text' => $this->t('Physical loans', [], ['context' => 'Dashboard']),
+      'physical-text' => $this->t('Physical', [], ['context' => 'Dashboard']),
+      'pick-up-latest-text' => $this->t('Pick up before', [], ['context' => 'Dashboard']),
+      'publizon-audio-book-text' => $this->t('Audiobook', [], ['context' => 'Dashboard']),
+      'publizon-ebook-text' => $this->t('E-book', [], ['context' => 'Dashboard']),
+      'publizon-podcast-text' => $this->t('Podcast', [], ['context' => 'Dashboard']),
+      'ready-for-loan-counter-label-text' => $this->t('Ready', [], ['context' => 'Dashboard']),
+      'ready-for-loan-modal-aria-description-text' => $this->t('This modal shows materials that are ready for loan', [], ['context' => 'Dashboard']),
+      'ready-for-loan-text' => $this->t('Ready for loan', [], ['context' => 'Dashboard']),
+      'ready-to-loan-close-modal-aria-label-text' => $this->t('Close ready to loan details modal', [], ['context' => 'Dashboard']),
+      'remove-all-reservations-text' => $this->t('Remove reservations', [], ['context' => 'Dashboard']),
+      'reservations-ready-text' => $this->t('Ready for you', [], ['context' => 'Dashboard']),
+      'reservations-still-in-queue-for-text' => $this->t('Still in queue', [], ['context' => 'Dashboard']),
+      'reservations-text' => $this->t('Reservations', [], ['context' => 'Dashboard']),
+      'result-pager-status-text' => $this->t('Showing @itemsShown out of @hitcount loans', [], ['context' => 'Dashboard']),
+      'still-in-queue-close-modal-aria-label-text' => $this->t('Close still in queue details modal', [], ['context' => 'Dashboard']),
+      'still-in-queue-modal-aria-description-text' => $this->t('This modal shows materials that are still in queue', [], ['context' => 'Dashboard']),
+      'total-owed-text' => $this->t('You owe in total', [], ['context' => 'Dashboard']),
+      'warning-icon-alt-text' => $this->t('warningIconAltText', [], ['context' => 'Dashboard']),
+      'your-profile-text' => $this->t('Your profile', [], ['context' => 'Dashboard']),
     ] + DplReactAppsController::externalApiBaseUrls();
 
     return [
-      "#theme" => "dpl_react_app",
-      "#name" => "DashBoard",
-      "#data" => $data,
+      '#theme' => 'dpl_react_app',
+      '#name' => 'DashBoard',
```

Comment:
Please use snake case in theme name. That is what we have done in other [places](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/64d430729cd35f5a7315ae68abc16bec7dae9ff9/web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php#L149).